handlers/multistage-diagram-tactile-svg/tactile_svg.py

```diff
@@ -0,0 +1,432 @@
+                coords = np.asarray(coords)
+                coords_max = coords.max(axis=0, keepdims=True)
+                coords_min = coords.min(axis=0, keepdims=True)
+                coord_vals = (coords_min[0][0]
+                              if (coords_min[0][0] < coord_vals[0])
+                              else coord_vals[0],
+                              coords_min[0][1]
+                              if (coords_min[0][1] < coord_vals[1])
+                              else coord_vals[1],
+                              coords_max[0][0]
+                              if (coords_max[0][0] > coord_vals[2])
+                              else coord_vals[2],
+                              coords_max[0][1]
+                              if (coords_max[0][1] > coord_vals[3])
+                              else coord_vals[3])
+            g.append(p)
+        coord_lims[stage["id"]] = {"lims": coord_vals, "name": label}
+    for link in data["links"]:
+        try:
+            label = ("Arrow between " + coord_lims[link["source"]]["name"] +
+                     " and " + coord_lims[link["target"]]["name"])
+            src_cntr = ((coord_lims[link["source"]]["lims"][2] +
+                        coord_lims[link["source"]]["lims"][0])/2,
+                        (coord_lims[link["source"]]["lims"][3] +
+                        coord_lims[link["source"]]["lims"][1])/2)
+            tgt_cntr = ((coord_lims[link["target"]]["lims"][2] +
+                        coord_lims[link["target"]]["lims"][0])/2,
+                        (coord_lims[link["target"]]["lims"][3] +
+                        coord_lims[link["target"]]["lims"][1])/2)
+            arw_src = cohenSutherlandClip(src_cntr, tgt_cntr,
+                                          (coord_lims
+                                           [link["source"]]["lims"]))
+            arw_tgt = cohenSutherlandClip(tgt_cntr, src_cntr,
+                                          (coord_lims
+                                           [link["target"]]["lims"]))
+            arw_src = point_at_distance(arw_src, arw_tgt, 0.1)
+            arw_tgt = point_at_distance(arw_tgt, arw_src, 0.2)
+            p = draw_arrow(arw_src, arw_tgt, dimensions,
+                           label, link["directed"])
+            g.append(p)
+        except Exception as e:
+            logging.debug("Encountered error while drawing arrow")
+            logging.pii(e.message)
+
+    svg.append(g)
+
+    # Checking if graphic-caption preprocessor is present
+    if ("ca.mcgill.a11y.image.preprocessor.graphic-caption"
+            in preprocessors):
+        logging.debug("Adding title from "
+                      "graphic-caption")
+        caption = preprocessors["ca.mcgill.a11y.image."
+                                "preprocessor.graphic-caption"][
+                            "caption"]
+    else:
+        logging.debug("graphic-caption not found. "
+                      "Adding default title.")
+
```

Comment:
You put this as a required dependency, but based on this code it appears to actually be optional.